regen/embed.pl

```diff
@@ -46,7 +46,7 @@ ($)
     $error_count++;
 }
 
-sub full_name ($$) { # Returns the function name with potentially the
+sub full_name { # Returns the function name with potentially the
```

Comment:
alternatively we could consider using `sub full_name :prototype($$)`

Reply:
Since I've never used prototypes at all, I'll have to trust you on that!  See 7aad926102